scripts/generate_proxy_paths.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python2
+
+import errno
+import json
+import os.path
+import re
+import shutil
+import sys
+
+if len(sys.argv) != 2:
+    sys.stderr.write("usage: %s /path/to/matrix-doc\n" % (sys.argv[0],))
+    sys.exit(1)
+
+outfile = os.path.abspath(os.path.join(os.path.dirname(os.path.dirname(__file__)), "src/github.com/matrix-org/dendron/proxy/paths.go"))
+
+templating_dir = os.path.join(sys.argv[1], "templating")
+sys.path.insert(0, templating_dir)
+os.chdir(templating_dir)
+
+from matrix_templates.units import resolve_references, MatrixUnits
+
+apis = MatrixUnits().load_swagger_apis()
+
+v2_paths = ["/account", "/register", "/sync", "/user"]
```

Comment:
You're missing a few v2 paths here, you might want to take a look at the output of running ``scripts-dev/list_url_patterns.py synapse`` in synapse